less/index.less

```diff
@@ -1 +1,13 @@
 // Follow the order in the readme
+
+@import "variables.less";
+@import "mixins.less";
+@import "reset.less";
+@import "global.less";
+@import "navigation.less";
+@import "footer.less";
+@import "home-page.less";
+@phone: ~"(min-width: 0px) and (max-width: 600px)";
```

Comment:
Good job and example of using escaping to write a `phone` variable for media queries, but it should be in the variables file not the index file. This file is just for importing the more specific less files.